grails-app/controllers/org/broadinstitute/orsp/UserController.groovy

```diff
@@ -29,4 +35,52 @@ class UserController {
         render syncedUsers as JSON
     }
 
+    def rolesManagement() {
+        render(view: "/rolesManagement/index")
+    }
+
+    def getOrspUsers() {
+        JSON.registerObjectMarshaller(User) {
+            def output = [:]
+            output['id'] = it.id
+            output['userName'] = it.userName
+            output['displayName'] = it.displayName
+            output['emailAddress'] = it.emailAddress
+            output['createdDate'] = it.createdDate
+            output['updatedDate'] = it.updatedDate
+            output['lastLoginDate'] = it.lastLoginDate
+            output['roles'] = it.roles.stream().map{role ->
+                role.getRole()}.collect()
```

Comment:
Can be simplified ... `it.roles.collect{r -> r.getRole()}` Groovy introduced just about everything that javas streams do long before java got around to it.

Reply:
Thanks for the suggestion! it's now simplified.